nifi-toolkit/nifi-toolkit-assembly/docker/tests/tls-toolkit.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
This file needs to be added to the excludes for the apache-rat-plugin in nifi-toolkit-assembly.